Adyen/Model/Checkout/PaymentMethodsRequest.cs

```diff
@@ -47,517 +47,134 @@ public partial class PaymentMethodsRequest :  IEquatable<PaymentMethodsRequest>,
+            this.ShopperLocale = shopperLocale;
+            this.ShopperReference = shopperReference;
+            this.ThreeDSAuthenticationOnly = threeDSAuthenticationOnly;
         }
-        
-        /// <summary>
-        /// Shopper account information for 3D Secure 2.0.
-        /// </summary>
-        /// <value>Shopper account information for 3D Secure 2.0.</value>
-        [DataMember(Name="accountInfo", EmitDefaultValue=false)]
-        public AccountInfo AccountInfo { get; set; }
-
-        /// <summary>
-        /// If you want a [BIN or card verification](https://docs.adyen.com/developers/payment-methods/cards/bin-data-and-card-verification) request to use a non-zero value, assign this value to &#x60;additionalAmount&#x60; (while the amount must be still set to 0 to trigger BIN or card verification). Required to be in the same currency as the &#x60;amount&#x60;. 
-        /// </summary>
-        /// <value>If you want a [BIN or card verification](https://docs.adyen.com/developers/payment-methods/cards/bin-data-and-card-verification) request to use a non-zero value, assign this value to &#x60;additionalAmount&#x60; (while the amount must be still set to 0 to trigger BIN or card verification). Required to be in the same currency as the &#x60;amount&#x60;. </value>
-        [DataMember(Name="additionalAmount", EmitDefaultValue=false)]
-        public Amount AdditionalAmount { get; set; }
 
         /// <summary>
-        /// This field contains additional data, which may be required for a particular payment request.  The &#x60;additionalData&#x60; object consists of entries, each of which includes the key and value. For more information on possible key-value pairs, refer to the [additionalData section](https://docs.adyen.com/developers/api-reference/payments-api#paymentrequestadditionaldata).
+        /// This field contains additional data, which may be required for a particular payment request.  The &#x60;additionalData&#x60; object consists of entries, each of which includes the key and value.
         /// </summary>
-        /// <value>This field contains additional data, which may be required for a particular payment request.  The &#x60;additionalData&#x60; object consists of entries, each of which includes the key and value. For more information on possible key-value pairs, refer to the [additionalData section](https://docs.adyen.com/developers/api-reference/payments-api#paymentrequestadditionaldata).</value>
-        [DataMember(Name="additionalData", EmitDefaultValue=false)]
-        public Dictionary<string, string> AdditionalData { get; set; }
+        /// <value>This field contains additional data, which may be required for a particular payment request.  The &#x60;additionalData&#x60; object consists of entries, each of which includes the key and value.</value>
+        [DataMember(Name = "additionalData", EmitDefaultValue = false)]
+        public AnyOfPaymentMethodsRequestAdditionalData AdditionalData { get; set; }
```

Comment:
Not sure how this new class will work. It was a map before

Reply:
Revert it. It was autogenerated but did not have a container to hold the additional data